scripts/external-dns-prerequisites/create_a_records.sh

```diff
@@ -0,0 +1,151 @@
+
+function createARecord() {
+    local RECORD_NAME=$1
+    local IP_ADDRESS=$2
+    printf "Creating A record with name $RECORD_NAME and IP $IP_ADDRESS..."
+    FIND_RECORD=$(az network dns record-set a show \
+        --name "$RECORD_NAME" \
+        --resource-group "$AZ_RESOURCE_GROUP_COMMON" \
+        --zone-name "$AZ_RESOURCE_DNS" \
+        --query name \
+        --output tsv \
+        2>/dev/null)
+    if [[ $FIND_RECORD = "" ]]; then
+        # Create A record
+        az network dns record-set a add-record \
+            --resource-group "$AZ_RESOURCE_GROUP_COMMON" \
+            --zone-name "$AZ_RESOURCE_DNS" \
+            --record-set-name "$RECORD_NAME" \
+            --ipv4-address "$IP_ADDRESS" \
+            --if-none-match \
+            --ttl 10 \
+            2>&1 >/dev/null
+    else
+        # Update A record
+        az network dns record-set a update \
+            --name "$RECORD_NAME" \
+            --resource-group "$AZ_RESOURCE_GROUP_COMMON" \
+            --zone-name "$AZ_RESOURCE_DNS" \
+            --set aRecords[0].ipv4Address="$IP_ADDRESS" \
+            2>&1 >/dev/null
+    fi
+}
+
+
+createARecord $CLUSTER_NAME $SELECTED_INGRESS_IP_RAW_ADDRESS || \
+  printf "ERROR: failed to create A record" >&2
+
+if [ "$MIGRATION_STRATEGY" = "aa" ]; then
+    old_ip=$(az network dns record-set a show --name active-cluster --zone-name $AZ_RESOURCE_DNS --resource-group $AZ_RESOURCE_GROUP_COMMON | jq .aRecords[0].ipv4Address --raw-output)
+    echo -e ""
+    echo -e "About to change active cluster DNS record"
+    echo -e ""
+    echo -e "   > WHERE:"
+    echo -e "   ------------------------------------------------------------------"
+    echo -e "   -  RADIX_ZONE                       : $RADIX_ZONE"
+    echo -e "   -  AZ_RESOURCE_DNS                  : $AZ_RESOURCE_DNS"
+    echo -e ""
+    echo -e "   > WHAT:"
+    echo -e "   -------------------------------------------------------------------"
+    echo -e "   -  RECORD_NAME                      : active-cluster"
+    echo -e "   -  OLD_IP                           : $old_ip"
+    echo -e "   -  NEW_IP                           : $SELECTED_INGRESS_IP_RAW_ADDRESS"
+    echo -e ""
+    echo -e "   > WHO:"
+    echo -e "   -------------------------------------------------------------------"
+    echo -e "   -  AZ_SUBSCRIPTION                  : $(az account show --query name -otsv)"
+    echo -e "   -  AZ_USER                          : $(az account show --query user.name -o tsv)"
+    echo -e ""
+
+    echo ""
+
+    if [[ $USER_PROMPT == true ]]; then
+        while true; do
+            read -r -p "Is this correct? (Y/n) " yn
+            case $yn in
+            [Yy]* ) break;;
+            [Nn]* ) echo ""; echo "Quitting."; exit 1;;
+            * ) echo "Please answer yes or no.";;
+            esac
+        done
+        echo ""
+    fi
+
+    createARecord active-cluster $SELECTED_INGRESS_IP_RAW_ADDRESS || \
+      printf "ERROR: failed to create A record" >&2
+fi
```

Comment:
```suggestion
    createARecord active-cluster $SELECTED_INGRESS_IP_RAW_ADDRESS || \
        printf "ERROR: failed to create A record" >&2
fi

```
indentation and newline